requirements.txt

```diff
@@ -1,3 +1,6 @@
+getpass
```

Comment:
Thanks for double checking, but these don't need to be added, as they are built-in python libraries: [re](https://docs.python.org/3/library/re.html), [uuid](https://docs.python.org/3/library/uuid.html), [sqlite3](https://docs.python.org/3/library/sqlite3.html), [getpass](https://docs.python.org/3/library/getpass.html). The requirements.txt only needs external libraries, not any created libraries by python.org. And unfortunately, if you run `pip install -r requirements.txt` with those in the file, it will throw an error. The purpose of `pip install` is to download those libraries, so that we can import them into our files, and pythons built-in libraries are already downloaded when you install python.